go.mod

```diff
@@ -2,13 +2,14 @@ module github.com/cloudnativelabs/kube-router
 
 require (
 	github.com/aws/aws-sdk-go v1.38.70
-	github.com/containerd/containerd v1.5.2 // indirect
+	github.com/containerd/containerd v1.5.5 // indirect
```

Comment:
It seems like your suggestions with 1.5.4 is the better choice. 1.5.5 updates protobuf which breaks linting as it deprecated some functions we use and you probably don' t want to update that one in this change.

Reply:
reverted